create_topomap_node:
  ros__parameters:
    topomap_images_dir: "../topomaps/images"
